pipx/commands.py

```diff
@@ -327,6 +335,71 @@ def install(
         venv.remove_venv()
         raise
 
+    # if all is well, write out pipxrc file
+    package_or_url_pipxrc = _abs_path_if_local(package_or_url, venv, pip_args)
+    pipxrc = Pipxrc(venv_dir, read=False)
+    pipxrc.set_package_or_url(package_or_url_pipxrc)
+    pipxrc.set_install_options(pip_args, venv_args, include_dependencies)
+    pipxrc.set_venv_metadata(venv.get_venv_metadata_for_package(package))
+    pipxrc.write()
+
+
+def _abs_path_if_local(package_or_url: str, venv: Venv, pip_args: List[str]) -> str:
```

Comment:
the body of this seems to do more than the function name implies; should we reformulate, split this up?

Reply:
The function name basically says what it does, if it determines the package is a local filepath, return the absolute path.  If it is not, then leave it alone.

Determining whether it is local or not definitively did turn out to be a little involved.  I also try to do quicker shortcuts first before I resort to checking pypi.

Reply:
I think it would be good to make a function out of the logic for computing `pip_search_found`, and corresponding unit tests.